.storybook/preview.js

```diff
@@ -1,4 +1,57 @@
 import { addDecorator } from '@storybook/react'
+import { StrictMode } from 'react'
 import { withPerformance } from 'storybook-addon-performance'
+import { createGlobalStyle } from 'styled-components'
 
-addDecorator(withPerformance)
+import { ResetCss } from './ResetCss'
+
+const GlobalStyle = createGlobalStyle`
+   @font-face {
+    font-family: 'Circular';
+    src: url('https://uploads-ssl.webflow.com/5baa461315ee32413d16236d/5f7f0665430aaedf8cb0bbf3_CircularXXSub-RegularSubset.woff2')
```

Comment:
I've used `createGlobalStyle` instead of putting it in `preview-head.html` as it is more similar to how we inject it in our app